src/pages/iou/IOUModal.js

```diff
@@ -121,6 +122,9 @@ class IOUModal extends Component {
             currentStepIndex: 0,
             participants: participantsWithDetails,
 
+            // Set default payment type to "Elsewhere"
+            paymentType: CONST.IOU.PAYMENT_TYPE.ELSEWHERE,
+
```

Comment:
@nickmurray47 What I am not sure is if this is a good pattern as I had to set the default `paymentType` in `IOUModal`, however, this need to be updated as if user has Expensify wallet the default payment should be the wallet.